packages/lesswrong/components/posts/TableOfContents/FixedPositionToC.tsx

```diff
@@ -15,6 +15,8 @@ import { usePostsPageContext } from '../PostsPage/PostsPageContext';
 import { SidebarsContext } from '../../common/SidebarsWrapper';
 import classNames from 'classnames';
 import { ToCDisplayOptions, adjustHeadingText, getAnchorY, isRegularClick, jumpToY } from './TableOfContentsList';
+import CommentsIcon from '@material-ui/icons/ModeComment';
```

Comment:
This should be replaced by using `ForumIcon` - we already have `Comment` defined using this icon there.